share/functions/type.fish

```diff
@@ -56,6 +56,15 @@ function type --description 'Print the type of a command'
                         end
                     case type
                         echo (_ 'function')
+                    case path
+                        set -l func_path (functions --detail $i)
```

Comment:
Wait... this supports "--detail" and "--details"? That's weird! The manpage says it's "--details".